widget/list.go

```diff
@@ -95,6 +95,19 @@ func (l *List) MinSize() fyne.Size {
 	return l.BaseWidget.MinSize()
 }
 
+func (l *List) moveFocusTo(id ListItemID) {
```

Comment:
This name confused me. Please don't use the word "Focus" when you are referring to visibility